cumulusci/cumulusci.yml

```diff
@@ -239,9 +239,9 @@ tasks:
         group: Salesforce DX
     dx_convert_from:
         description: Converts force-app directory in sfdx format into metadata format under src
-        class_path: cumulusci.tasks.sfdx.SFDXBaseTask
+        class_path: cumulusci.tasks.dx_convert_from.DxConvertFrom
         options:
-            command: "force:source:convert -d src"
+            command: "force:source:convert"
```

Comment:
I don't think we can make this change to add a separate src_dir option in a backwards-compatible way. Someone may have put this in a project cumulusci.yml:

```
tasks:
    dx_convert_from:
        options:
            command: "force:source:convert -d custom_path"
```

and with the new implementation they would end up with 2 paths specified in the sfdx command.

Reply:
Ah, but now I see how it has to be separate in order to easily find it so we know what path to delete. Okay, let's go ahead and make that change but be sure to call it out in the critical changes section of the release notes.

Reply:
@Br4nd0R we should remove the `command` option here